filament/backend/src/vulkan/VulkanDriver.cpp

```diff
@@ -609,6 +611,8 @@ void VulkanDriver::createRenderTargetR(Handle<HwRenderTarget> rth,
         depthStencil[0] = {
             .texture = mResourceAllocator.handle_cast<VulkanTexture*>(depth.handle),
             .level = depth.level,
+            .baseViewIndex = depth.baseViewIndex,
+            .layerCount = layerCount,
```

Comment:
Let's also add an `assert_invariant` here to make sure that layerCount matches `depth.layerCount`